experimenter/experimenter/experiments/models.py

```diff
@@ -746,6 +746,12 @@ def rollout_monitoring_dashboard_url(self):
         if self.is_rollout and (
             self.status
             in (NimbusExperiment.Status.LIVE, NimbusExperiment.Status.COMPLETE)
+            and (
+                self.end_date is None
+                or self.end_date
+                > datetime.date.today()
+                - datetime.timedelta(days=settings.ROLLOUT_MONITORING_EXPIRATION_DAYS)
```

Comment:
Can you add some order of operations parens around this, it's confusing to read 🙏